src/space.cpp

```diff
@@ -17,12 +17,11 @@
  * @license GPL v2+
  */
 
-#include "space.h"
-
 #include "add_space_table.h"
 #include "log_rules.h"
 #include "options_for_QT.h"
 #include "punctuators.h"
+#include "space.h"
```

Comment:
Leave the include at the old place, before the block of many includes.

Reply:
Yep got it, sorry for that :sweat_smile: